weave-js/src/common/hooks/useViewerUserInfo.ts

```diff
@@ -34,7 +34,7 @@ export const useViewerUserInfo = (): UserInfoResponse => {
 };
 
 const VIEWER_QUERY = gql`
-  query Viewer {
+  query Viewer2 {
```

Comment:
Definition was conflicting after changes in useViewerInfo - need to combine these files.